### ISSUE TITLE
Update mermaid to v10.9.1

### ISSUE THREAD

**Issue by AlixANNERAUD** [jupyter/nbconvert#2151]
Hello,
would it be possible to update Mermaid to version v10.9.1?
The introduction of the "block diagram" feature in version v10.8.0 would be very useful with nbconvert.
Additionally, would it be possible to manually specify the Mermaid version in the configuration file if needed (for example: `c.SlidesExporter.font_awesome_url`).
Thank you for this great project.

**Comment by franzhaas** [comment]
Hi, 

just out of curiosity, for you mermaid does work? I ended up needing this change,

https://github.com/jupyter/nbconvert/pull/2120

just curious,
Franz

**Comment by AlixANNERAUD** [comment]
Yes, it works, but only if nbconvert is installed via pip, not through the official Ubuntu/Debian repositories.
